docs/library/urandom.rst

```diff
@@ -0,0 +1,49 @@
+:mode:`urandom` -- generates random numbers
+===========================================
+
+.. module:: urandom
+   :synopsis: random numbers
+
+This module implements pseudo-random number generator. The functions that
+these module provides are bond methods of a hidden number of :random.Random:
+class.   
+
+Given below are the methods that this module provides.
+
+  .. method:: urandom 
+
+    Returns an n-bit pseudo-random number(output is determinsitic).
+
+  .. method:: urandom.getrandbits(n)
+
+    Returns an integer with 'n' random bits (1<=n<=32)
+
+  .. method:: urandom.seed(n)
```

Comment:
```suggestion
  .. method:: urandom.seed(n=None)
```
The `None` case only works if `MICROPY_PY_URANDOM_SEED_INIT_FUNC` is defined in the port, otherwise raises `ValueError`.